docs/createwebhook.md

```diff
@@ -1,16 +1,163 @@
-# Create Webhook Tekton Task
-The create webhook task configures necessary componetns for the github webhook sending event to the event listener.
-It configures following compoments:
-
-1. The github webhook
-1. Tekton event listener
-1. Ingress for the event listener
-1. Selfsigned certificate for the ingress
+# Create GitHub Webhook Tekton Task
+The following `Task` allows you to easily and securely configure a GitHub webhook against an `EventListener`:
+
+<!-- FILE: docs/create-webhook.yaml -->
+```YAML
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: create-webhook
+spec:
+  inputs:
+    params:
+    - name: CreateCertificate
+      description: "Enables/disables the creation of the self-signed certificate for the Ingress"
+      default: "true"
+    - name: CreateIngress
+      description: "Enables/disables the creation of the Ingress for the EventListener"
+      default: "true"
+    - name: CreateWebhook
+      description: "Enables/disables the creation of the webhook configuration in the GitHub repo"
+      default: "true"
+    - name: CreateEventListener
+      description: "Enables/disables the creation of the EventListener"
+      default: "true"
+    - name: CertificateKeyPassphrase
+      description: "Phrase that protects private key. This must be provided when the self-signed certificate is created."
+      default: "phrase1"
+    - name: CertificateSecretName
+      description: "Secret name for ingress certificate. The secret should not exist if the self-signed certificate creation is enabled."
+    - name: ExternalURL
+      description: "The external access URL for the EventListener."
+      default: "https://$(inputs.params.EventListenerName).<PROXYIP>.nip.io"
```

Comment:
Please add to the description that `ExternalURL` needs to be a valid IP address. Also, the `default` value here is not valid; plus, I don't think you can reference `$(inputs.params)` from inside the default param (but I could be wrong)

Reply:
Yeah, I think this is the only instance where the default is a valid default since `<PROXYIP>` wouldn't be known? I could push this into the description?

Reply:
The `<PROXYIP>` part is fine, but it shouldn't have `https://`. In the Task, it's used as `https://$(inputs.params.ExternalURL)`, so we don't want a double `https://`.